backend/src/routers/gameRouter.ts

```diff
@@ -53,6 +57,45 @@ const addGameSchema = z.object({
 	playtime: z.number().int().min(1)
 });
 
+/**
+ * @api {get} /api/v1/games/search/:term Request Games
```

Comment:
This is no longer accurate, as the code has been updated to instead use `/api/v1/games/search` with a query parameter of term, like this `/api/v1/games/search?term=something`. The docs should be updated accordingly